discord/ext/commands/errors.py

```diff
@@ -1138,15 +1138,23 @@ class BadFlagArgument(FlagError):
     -----------
     flag: :class:`~discord.ext.commands.Flag`
         The flag that failed to convert.
+    argument: :class:`str`
+        The argument supplied by the caller that was not able to be converted.
+    original: :class:`Exception`
+        The original exception that was raised. You can also get this via
+        the ``__cause__`` attribute.
     """
 
-    def __init__(self, flag: Flag) -> None:
+    def __init__(self, flag: Flag, argument: str, original: Exception) -> None:
         self.flag: Flag = flag
         try:
             name = flag.annotation.__name__
         except AttributeError:
             name = flag.annotation.__class__.__name__
 
+        self.argument = argument
+        self.original = original
```

Comment:
```suggestion
        self.argument: str = argument
        self.original: Exception = original
```

Reply:
Wanted to have a way to get the flag and input that failed since they're not accessible from ctx